lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -128,8 +128,8 @@ def scrub(transcript)
           gsub(%r((oauth_nonce=\")\w+), '\1[FILTERED]').
           gsub(%r((oauth_signature=\")[a-zA-Z%0-9]+), '\1[FILTERED]').
           gsub(%r((oauth_token=\")\w+), '\1[FILTERED]').
-          gsub(%r((number\D+)\d{16}), '\1[FILTERED]').
-          gsub(%r((cvc\D+)\d{3}), '\1[FILTERED]').
+          gsub(%r((number\\\":\\\")\d+), '\1[FILTERED]').
+          gsub(%r((cvc\\\":\\\")\d+), '\1[FILTERED]').
```

Comment:
Hmm, why did we change this to be so specific? Was the `\D+` that would include any non digit characters causing issues?

Reply:
I'm just a little concerned that if there is white space or something that it will stop scubbing the digits.

Reply:
In remote test this is exactly how quickbooks returns it and it scrubs properly. The reason I change it was because `cvc\D+` was scrubbing other random ids in the response.

Reply:
Having it like `(cvc\D+)\d+` was scrubbing single digits

Reply:
Ah that makes sense. I'm good with this!